app/code/Magento/Sales/Block/Adminhtml/Order/Invoice/View.php

```diff
@@ -113,8 +113,8 @@ protected function _construct()
                 $orderPayment->canRefund() && !$this->getInvoice()->getIsUsedForRefund()
             ) {
                 $this->buttonList->add(
-                    'capture',
-                    [ // capture?
+                    'creditmemo',
```

Comment:
`credit-memo`, please. And amend commit.

Reply:
@orlangur done